frontend/src/components/static_pages/about.css

```diff
@@ -87,8 +90,29 @@
     font-size: 16px;
   }
   .about-heading {
-    font-size: 32px;
+    font-size: 45px;
     margin-left: 0;
+    text-align: center;
+  }
+  .people-col {
+    display: inline-block;
+    margin-left: 20px;
+  }
+  .people-heading {
+    font-size: 40px;
+    margin-top: 20px;
+  }
+  .people-col .sub-heading {
+    padding-top: 7px;
+  }
+  .about-people-row {
+    margin: 0px;
+  }
+  #shirley {
+    margin: 40px 0px 30px 0px;
+  }
+  #pierre {
+    margin-top: 20px;
```

Comment:
Why the difference in spacing for #pierre and #shriley? I would recommend consistent spacing for all of the people rows.

Reply:
I think that I did the changes like you suggested but let me know if there are more things to work out on....Thank you